### ISSUE TITLE
do i use 

### ISSUE THREAD

**Issue by teknotopya** [renzuzu/renzu_shops#119]
i want to use it on scripting qbox. therefore, is it possible for me to use the inventory as ox with framework as qb (qbox)?
cross platform like..

**Comment by renzuzu** [comment]
yes, set the framework to qbcore and set the inventory to ox-inventory

edit /init.lua
